src/external/Collector.ts

```diff
@@ -546,7 +546,7 @@ class Collector extends BaseExternal {
       gas: '0x' + txObj.gasLimit.toString(16),
       value: '0x' + txObj.value.toString('hex'),
       input: '0x' + txObj.data.toString('hex'),
-      gasPrice: '0x' + txObj.getBaseFee().toString(16),
```

Comment:
In the existing code:

`getBaseFee()` returns the min gas the tx must have, not the gas price.

![image](https://github.com/user-attachments/assets/fe0dc78e-a24c-4fd6-8c91-bd42e680b245)